CHANGES.md

```diff
@@ -10,6 +10,8 @@ Change Log
    * `Cesium3DTileContent`
    * `Cesium3DTileFeature`
    * `Cesium3DTilesInspector`, `Cesium3DTilesInspectorViewModel`, and `viewerCesium3DTilesInspectorMixin`
+   * `Cesium3DTileColorBlendMode`
+   * `Cesium3DTileRefine`
```

Comment:
Remind me why this is public?  Because it is on a getter in `Cesium3DTile`?  What is the use case?